templates/students/update_profile.html

```diff
@@ -28,31 +28,24 @@
        opacity: 1;
 
     }
-    .caption {
-       background-color: rgba(128,128,128,0.5);
-       color: white;
-       font-size: 30px;
-       padding-top:30%;
-       border-radius: 6px;
-       height: 100%;
-       font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-       font-weight: bolder;
-       z-index: 10;
-    }
-    .hoverText {
-       position: absolute;
-       top:0;
-       height: 150px;
-       transition: .5s ease;
-       opacity: 0;
-       width: 150px;
-       text-align: center;
-    }
-    #container {
-      width: 100px;
-      height: 100px;
-      position: relative;
-}
+    
+  #profile-image-popover{
+  position: relative;
+  }
+  #profile-image-popover:after{
+  position: absolute;
```

Comment:
I think you've missed the dark overlay over the image. Same can be implemented as you did it before.
(You can fix the square overlay by using `border-radius: 50%;`, if that's creating a problem for you.)

Reply:
Didn't get this.. Can you explain little further

Reply:
The dark background (overlay) over the image, which should appear on hovering over the image.